crates/bevy_render/src/mesh/mesh.rs

```diff
@@ -464,7 +464,7 @@ impl Mesh {
         let normals: Vec<_> = positions
             .chunks_exact(3)
             .map(|p| face_normal(p[0], p[1], p[2]))
-            .flat_map(|normal| std::array::IntoIter::new([normal, normal, normal]))
+            .flat_map(|normal| [normal, normal, normal].into_iter())
```

Comment:
why is the `into_iter` call necessary, wouldn't it just work to return the array?

Reply:
Removed `into_iter` call